src/main/java/org/example/dao/PointDAOImpl.java

```diff
@@ -0,0 +1,80 @@
+package org.example.dao;
+
+import org.example.model.Point;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PointDAOImpl implements PointDAO {
+    private Connection connection;
+
+    public PointDAOImpl() {
+        try {
+            connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/DB", "username", "password");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void insertPoint(Point point) {
+        try {
+            PreparedStatement statement = connection.prepareStatement("INSERT INTO point (id, xCoordinate, yCoordinate) VALUES (?, ?, ?)");
+            statement.setInt(1, point.getId());
```

Comment:
Please use auto_generated keys

Reply:
done

Reply:
I don't think we're supposed to have the statement.setInt(1, point.getId()); there still, but we can change that after. Just a note for future reference